apps/camera/js/views/hud.js

```diff
@@ -40,14 +40,37 @@ module.exports = View.extend({
     return this;
   },
 
+  _setLabel: function(element, mode) {
+    if (mode) {
+      this.els[element].setAttribute('data-l10n-id', mode.icon + '-button');
```

Comment:
Seems brittle here to use the icon name to construct the `data-l10n-id` attribute. If the icon changes this will break your localization.